### DISCUSSION TITLE
Automated tag categories

### DISCUSSION THREAD

**Discussion by deepseawitch** [shish/shimmie2#1215] [Q&A] [answered]
First of all sorry if this is the wrong place to ask, I'm not used to posting on Github. I wanted to ask if it's possible to make it so that tag categories are automated on the Shimmie instance we're running. Right now we have it so that when you type "series:(name)" it creates a tag for the series. What we want is that simply typing the series name automatically adds the series tag (like on Danbooru / Gelbooru. e.g: adding simply "touhou" to an image would add the "touhou" series tag instead of having to type "series:touhou" every time a touhou image is submitted). Thanks in advance.

**Comment by discomrade** [comment, marked as answer]
You might want to use tag aliases (under Tags menu), adding `touhou` as an alias of `series:touhou`. I assume that's also what you're talking about on Danbooru / Gelbooru. This will need to be done for each series, character, etc..

Adding an alias does change existing tags, so always make sure there are no double-meanings or exceptions for an alias **before** you add it. And, hopefully, you're keeping regular database backups anyway.

**Reply by deepseawitch** [reply]
I was aware of the alias method but I thought there'd be one that doesn't require it since I've seen other shimmie instances that do this. Either way, thank you.

**Reply by discomrade** [reply]
>since I've seen other shimmie instances that do this

Can you please link an example? I don't really understand the feature, if it's different to aliases.

**Reply by deepseawitch** [reply]
[Gyate](https://gyate.net/) is the only one I can think of right now which has this feature. I've seen others, which I sadly cannot remember right now.
